spec/test-utils.tsx

```diff
@@ -51,11 +51,32 @@ const delay = (ms) =>
 const getAttribute = (container) => (attribute) =>
   container.querySelector(`[${attribute}]`)?.getAttribute(attribute);
 
+function transformSalePrice(item, mockSalePrice) {
+  const itemCopy = { ...item };
+  if (itemCopy.variations && Array.isArray(itemCopy.variations)) {
+    itemCopy.variations = itemCopy.variations.map((variation) => ({
```

Comment:
I'm curios why you're copying the item and the variations to add the `sale_price` attribute
How about iterating through the `variations` and just assigning the mockSalePrice inside?
Something like
```
itemCopy.variations.forEach((variation) => {
  variation.data.sale_price = mockSalePrice
})
```

If the intention of that function is to also copy the item as to not mess with the original data, I guess we should suggest a better naming? what do you think?